searchcore/src/vespa/searchcore/proton/attribute/attributemanager.cpp

```diff
@@ -145,7 +145,7 @@ AttributeManager::internalAddAttribute(const AttributeSpec &spec,
 void
 AttributeManager::addAttribute(const AttributeWrap &attribute, const ShrinkerSP &shrinker)
 {
-    LOG(debug, "Adding attribute vector '%s'", attribute.getAttribute()->getBaseFileName().c_str());
+    LOG(debug, "Adding attribute vector '%s'", attribute.getAttribute()->getName().c_str());
```

Comment:
Too little information, document type and document subdb name cannot be deduced from log message.